hub/scripts.js

```diff
@@ -415,8 +415,17 @@
    * Page details are updated based on these values.
    */
   function handlePageDetails() {
-    const paths = window.location.pathname.replace('/bench', '')
+    const path = window.location.pathname;
+    let paths;
+
+    if (path.indexOf('_draft')) {
```

Comment:
I'm checking first if there is draft in the path, then removing it.
Do you have any suggestion to shorten if 
instead of doing, this one twice?
```
paths = path.replace('/bench' && '_draft', '')
      .split('/').filter(isNonEmptyString);
```

Reply:
hi @elaineskpt - you can chain methods and write it like `String.replace(x, '').replace(y, '')`;